cmd/storj-sim/network.go

```diff
@@ -25,8 +25,56 @@ import (
 	"storj.io/storj/internal/processgroup"
 )
 
+type peerClass int
+
+func (p peerClass) String() string {
+	return []string{"Satellite", "Version Control", "Boostrap", "Gateway", "Storage Node"}[p]
+}
+
+const (
+	// The following values of peer class, index, and endpoints are used
+	// to create a port with a consistent format for storj-sim services.
+
+	// Peer class
+	satellitePeer peerClass = iota
```

Comment:
Use explicit values here instead of iota. When defining constants that should be preserved from version to version, then having explicit values makes it more obvious.